scheduler/views.py

```diff
@@ -22,13 +22,18 @@ class CreatedAtCursorPagination(CursorPagination):
     ordering = "-created_at"
 
 
+class IdCursorPagination(CursorPagination):
+    ordering = "-id"
```

Comment:
I think maybe we can just sort in the forward `id` direction here? Since I think the default index for id is in the forward direction.